index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>SURVEY PAGE</title>
+</head>
+<body>
+    <link rel="stylesheet" href="style.css">
+
+<div class="header">
+    <h1 id="title"> Our Survey Form</h1>
+    <p>Thank you for taking the time to help us improve the platform</p>
+</div>
+
+    <div class="row">
+    <p id="description">Kenny and James survey-form :)</p>
+    <form action="web url" id="survey-form">
+    </form>
```

Comment:
Please add the actual url link to the action attribute that will be directing the user input to a server.